src/v/pandaproxy/schema_registry/requests/acls.cc

```diff
@@ -47,7 +47,24 @@ T parse_security_enum(std::string_view value, std::string_view type_name) {
 } // namespace
 
 security::acl_operation to_acl_operation(std::string_view op) {
-    return parse_security_enum<security::acl_operation>(op, "ACL operation");
+    auto res = parse_security_enum<security::acl_operation>(
+      op, "ACL operation");
+    switch (res) {
+    case security::acl_operation::all:
+    case security::acl_operation::read:
+    case security::acl_operation::write:
+    case security::acl_operation::remove:
+    case security::acl_operation::describe:
+    case security::acl_operation::describe_configs:
+    case security::acl_operation::alter:
+    case security::acl_operation::alter_configs:
+        break;
+    default:
+        throw exception(
+          error_code::acl_invalid,
+          fmt::format("Invalid acl operation: {}", op));
+    }
+    return res;
```

Comment:
Ahh. I think maybe we could extract and reuse this? https://github.com/redpanda-data/redpanda/blob/v25.2.1-rc2/src/v/kafka/server/handlers/details/security.h#L369-L383 - I say reuse, but that's probably dead code.